hikari/impl/event_manager.py

```diff
@@ -62,9 +60,6 @@
     from hikari.internal import data_binding
 
 
-_LOGGER: typing.Final[logging.Logger] = logging.getLogger("hikari.event_manager")
```

Comment:
Would this debug logging not still be helpful for seeing how these events are being treated

Reply:
For now its not being used. When I pick this pr back (after exams), I will revert some of the changes and update the PR